utils/log/loggers/fields.go

```diff
@@ -12,6 +13,10 @@ var (
 
 //LogFields contains all fields that have to be added to logs
 type LogFields map[string]interface{}
+type ProtectedLogFields struct {
+	Content LogFields
+	mtx     sync.RWMutex
```

Comment:
is testing from `loggers_test` package the only reason to expose this struct? if so, what's the downside to write test in the same package?

the same for `Content`, if we write test from the same package, there is no need to expose this field and this keeps your encapsulation better.

Reply:
You're right, there's no need to expose these structs, changed and thanks.